### PR TITLE
Bitacora: Edit sidebar markup to use aside

### PR DESCRIPTION
**Pull request by iamtakashi**
#### Changes proposed in this Pull Request:
Change sidebar markup to use aside for better accessibility. Ref: pdXvLu-1X2-p2

### CODE REVIEW

File: bitacora/readme.txt [hunk 1]
```diff
@@ -12,6 +12,9 @@ Bitácora is a simple old-school blog theme.
 
 == Changelog ==
 
+= 1.0.8 =
+* Edit sidebar markup to use aside (#7278)
+
```
Comment (mikachan) [inline bitacora/readme.txt]: The changelog will be updated automatically when the theme is deployed, so we don't need to add a manual update here.

File: bitacora/style.css [hunk 2]
```diff
@@ -7,7 +7,7 @@ Description: Bitácora is a simple old-school blog theme.
 Requires at least: 6.0
 Tested up to: 6.1.1
 Requires PHP: 5.7
-Version: 1.0.7
+Version: 1.0.8
```
Comment (mikachan) [inline bitacora/style.css]: This will also be updated automatically on deployment.

**Review by mikachan — APPROVED** [review]
Thanks for this update! It looks good to me.

Just left a few comments about the changelog and version updates, as these don't need to be included in the PR.

File: bitacora/readme.txt (hunk 1 above)
Comment (iamtakashi) [reply bitacora/readme.txt]: Sorry about that!

File: bitacora/style.css (hunk 2 above)
Comment (iamtakashi) [reply bitacora/style.css]: I'll fix them now. Sorry!

**Comment by iamtakashi** [comment]
@mikachan I've revert the version bump that wasn't necessary. Thanks for letting me know about it.

**Comment by mikachan** [comment]
No worries, LGTM!

**Pull request merged**